fs.go

```diff
@@ -794,6 +795,10 @@ func (h *fsHandler) handleRequest(ctx *RequestCtx) {
 	if !ok {
 		pathStr := string(path)
 		filePath := h.root + pathStr
+		switch runtime.GOOS {
+		case "windows":
+			filePath = filepath.FromSlash(filePath)
```

Comment:
You don't need to check `runtime.GOOS` if you use `filepath.FromSlash`. It will take care of that.